keyboards/cannonkeys/link/keyboard.json

```diff
@@ -0,0 +1,141 @@
+{
+    "manufacturer": "CannonKeys",
+    "keyboard_name": "Link",
+    "maintainer": "awkannan",
+    "bootloader": "rp2040",
+    "bootmagic": {
+        "matrix": [0, 5]
+    },
+    "build": {
+        "debounce_type": "asym_eager_defer_pk"
+    },
+    "diode_direction": "COL2ROW",
+    "dynamic_keymap": {
+        "layer_count": 6
+    },
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP12", "pin_b": "GP13", "resolution": 2}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "oled": true
+    },
+    "matrix_pins": {
+        "cols": ["GP17", "GP18", "GP19", "GP20", "GP21", "GP24"],
+        "rows": ["GP25", "GP26", "GP16", "GP11", "GP10"]
+    },
+    "processor": "RP2040",
+    "qmk": {
+        "tap_keycode_delay": 10
+    },
+    "split": {
+        "bootmagic": {
+            "matrix": [5, 5]
+        },
+        "enabled": true,
+        "encoder": {
+            "right": {
+                "rotary": [
+                    {"pin_a": "GP12", "pin_b": "GP13", "resolution": 2}
+                ]
+            }
+        },
```

Comment:
Shouldn't need to specify this again, as it's the same as the left half.
```suggestion
```

Reply:
Just to be sure - this is fine to remove even if there are 2 encoder? (one left and one right)

Reply:
(I made the change)